README.md

```diff
@@ -147,61 +164,35 @@ containerize.  It is recommended that most users use a version tag (e.g.
 |`cisagov/example:latest`| The most recent release image pushed to a container registry.  Pulling an image using the `:latest` tag [should be avoided.](https://vsupalov.com/docker-latest-tag/) |
 
 See the [tags tab](https://hub.docker.com/r/cisagov/example/tags) on Docker
-Hub for a list of all the supported tags.
+Hub for a list of all the supported tags. -->
 
 ## Volumes ##
 
 | Mount point | Purpose        |
 |-------------|----------------|
-| `/var/log`  |  Log storage   |
-
-## Ports ##
-
-The following ports are exposed by this container:
-
-| Port | Purpose        |
-|------|----------------|
-| 8080 | Example only; nothing is actually listening on the port |
-
-The sample [Docker composition](docker-compose.yml) publishes the
-exposed port at 8080.
+| `/home/cisa`  |  Log storage   |
 
 ## Environment variables ##
 
 ### Required ###
 
-There are no required environment variables.
-
-<!--
 | Name  | Purpose | Default |
 |-------|---------|---------|
-| `REQUIRED_VARIABLE` | Describe its purpose. | `null` |
--->
+| `PCA_GENERATOR_IMAGE` | Docker image name. | `cisagov/pca-report-generator` |
 
-### Optional ###
+<!-- ### Optional ### -->
 
-| Name  | Purpose | Default |
+<!-- | Name  | Purpose | Default |
 |-------|---------|---------|
-| `ECHO_MESSAGE` | Sets the message echoed by this container.  | `Hello World from Dockerfile` |
+| `ECHO_MESSAGE` | Sets the message echoed by this container.  | `Hello World from Dockerfile` | -->
 
 ## Secrets ##
 
 | Filename     | Purpose |
 |--------------|---------|
 | `quote.txt` | Replaces the secret stored in the example library's package data. |
 
-## Building from source ##
-
-Build the image locally using this git repository as the [build context](https://docs.docker.com/engine/reference/commandline/build/#git-repositories):
-
-```console
-docker build \
-  --build-arg VERSION=0.0.1 \
-  --tag cisagov/example:0.0.1 \
-  https://github.com/cisagov/example.git#develop
-```
-
-## Cross-platform builds ##
+<!-- ## Cross-platform builds ##
```

Comment:
These sections should not have been removed/commented out.

Reply:
Resolved in 598aceef591cfd34045b8d82f810e36563a5925d